application-logging/esm-logs-in-context/compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: "3.0"
+
+services:
+  logs-in-context-esm-app:
+    container_name: nr-legacy-logs-in-context-esm-app
+    env_file: .env
+    environment:
+      NODE_ENV: production
+    build:
+      context: app
+    volumes:
+      - "./logs:/app/logs"
```

Comment:
is this needed?

Reply:
No. It was needed for streaming logs to a file, like the log gen example, but not needed here.